static/js/dashboard.js

```diff
@@ -166,7 +166,11 @@ dkron.controller('JobListCtrl', function ($scope, $http, $interval, hideDelay) {
 
   // calculate page in place
   $scope.groupToPages = function () {
-    $scope.gap = Math.round($scope.jobs.length / $scope.itemsPerPage);
+    if (Math.round($scope.jobs.length / $scope.itemsPerPage) < 10) {
+      $scope.gap = Math.round($scope.jobs.length / $scope.itemsPerPage);
+    } else {
+      $scope.gap = 10;
+    }
```

Comment:
$scope.gap = Math.round(....
$scope.gap = Math.min($scope.gap, 10);

Saves duplication of the gap calculation and an if.

Reply:
Thanks for that!